bin/init.sh

```diff
@@ -5,6 +5,14 @@ set -o nounset
 set -o pipefail
 set -x
 
+detected_OS=`uname -s 2>/dev/null || echo not`
```

Comment:
I think we don't really need the entire build here. `fetch` is sufficient. I can try to see whether that works later.

Reply:
comment on the wrong file? :)..